oscarlab/graphene
# [SGX Driver] Fix make clean on 5.8

## Description of the changes 

This is the companion to https://github.com/oscarlab/graphene-sgx-driver/pull/25

The last PR fixed the gitignore test, but was not enough to pass a clean check for the sgx driver on a 5.8 kernel.  This fix is sufficient.

## How to test this PR? 

Run the make clean check, or a jenkins pipeline on a 5.8 kernel.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1659)
